saleor/webhook/tests/test_webhook_payloads.py

```diff
@@ -43,108 +43,424 @@
         reason="Discount from staff",
     )
-    order_with_lines.discounts.create(
+    order.discounts.create(
         type=OrderDiscountType.VOUCHER,
         value_type=DiscountValueType.PERCENTAGE,
         value=Decimal("10"),
         amount_value=Decimal("16.5"),
         name="Voucher",
     )
 
-    line_without_sku = order_with_lines.lines.last()
+    line_without_sku = order.lines.last()
     line_without_sku.product_sku = None
     line_without_sku.save()
 
-    assert fulfilled_order.fulfillments.count() == 1
-    fulfillment = fulfilled_order.fulfillments.first()
+    return order
 
-    order_id = graphene.Node.to_global_id("Order", order_with_lines.id)
-    payload = json.loads(generate_order_payload(order_with_lines))[0]
 
-    assert order_id == payload["id"]
-    non_empty_fields = [f for f in ORDER_FIELDS if f != "collection_point_name"]
-    for field in non_empty_fields:
-        assert payload.get(field) is not None
+@pytest.fixture
+def payment_for_payload(payment_txn_captured):
+    payment_txn_captured.psp_reference = "123"
+    payment_txn_captured.save(update_fields=["psp_reference"])
+    return payment_txn_captured
 
-    assert payload["collection_point_name"] is None
-    assert payload.get("token") == order_with_lines.token
-    assert payload.get("shipping_method")
-    assert payload.get("shipping_tax_rate")
-    assert payload.get("lines")
-    assert payload.get("shipping_address")
-    assert payload.get("billing_address")
-    assert payload.get("fulfillments")
-    assert payload.get("discounts")
-    assert payload.get("original") == graphene.Node.to_global_id("Order", new_order.pk)
-    assert payload.get("payments")
-    assert payload.get("included_taxes_in_prices") == include_taxes_in_prices()
-    assert len(payload.get("payments")) == 1
-    payments_data = payload.get("payments")[0]
-    assert payments_data == {
-        "id": graphene.Node.to_global_id("Payment", payment_txn_captured.pk),
-        "gateway": payment_txn_captured.gateway,
-        "payment_method_type": payment_txn_captured.payment_method_type,
-        "partial": False,
-        "cc_brand": payment_txn_captured.cc_brand,
-        "is_active": payment_txn_captured.is_active,
-        "created": ANY,
-        "modified": ANY,
-        "charge_status": payment_txn_captured.charge_status,
-        "psp_reference": payment_txn_captured.psp_reference,
-        "total": str(payment_txn_captured.total.quantize(Decimal("0.01"))),
-        "type": "Payment",
-        "captured_amount": str(
-            payment_txn_captured.captured_amount.quantize(Decimal("0.01"))
+
+@freeze_time()
+@pytest.mark.parametrize("taxes_included", [True, False])
+@mock.patch("saleor.webhook.payloads._generate_order_lines_payload_with_taxes")
+@mock.patch("saleor.webhook.payloads.generate_fulfillment_lines_payload")
+def test_generate_order_payload(
+    mocked_fulfillment_lines,
+    mocked_order_lines,
+    mocked_fetch_order,
```

Comment:
do you still needed it?

Reply:
Yes, I want to make sure that we're sending fresh prices in order payload